graphql/tests/query/adminSettings/queries/announcements.graphql

```diff
@@ -1,4 +1,4 @@
-query AdminSettingsPermissionGranted {
+query Announcements {
```

Comment:
ah this makes sense i'll keep that in mind for future test i write

Reply:
dw it made sense at the time when you added it!